### PR TITLE
Repo maintenance

### PR DESCRIPTION
**Pull request by wow-such-code**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,61 @@
-# Experimental Design Library
+experimental-design-lib
+-----------------------------------
+# <p align=center>Experimental Design Library</p>
```
Comment (KochTobi) [inline README.md:3]: You only need one centered `div`.
https://github.com/qbicsoftware/sample-tracking-service/blob/master/README.md

File: README.md [hunk 2]
```diff
@@ -1,19 +1,61 @@
...
+```
+
+Then include this library as an artifact.
+```xml
+<dependency>
+    <groupId>life.qbic</groupId>
+    <artifactId>experimental-design-lib</artifactId>
+    <version>[version-number]</version>
+</dependency>
+```
```
Comment (KochTobi) [inline README.md:60]: Very nice readme. I am missing a description on the data model as well as what exactly this library aims to provide.
Maybe an example how to use this library to parse experimental properties to openbis

**Review by KochTobi — CHANGES_REQUESTED** [review]

File: .github/workflows/create-release.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+      - name: Open PR with version bump
+        uses: actions/github-script@v4.0.2
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            await github.request(`POST /repos/${{ github.repository }}/pulls`, {
+              title: 'Update version to ${{ github.event.inputs.versionTag }}',
+              head: 'release/set-version-to-${{ github.event.inputs.versionTag }}',
+              base: 'main'
```
[the remark this answers is not in the document]
Comment (wow-such-code) [reply .github/workflows/create-release.yml:92]: good catch!

File: src/test/java/life/qbic/expdesign/io/MetaboDesignReaderTest.java [hunk 4]
```diff
@@ -29,8 +29,8 @@
 
   private File fullExample1 =
       new File(getClass().getResource("mtx/metabo_full_example1.tsv").getFile());
-  private File withoutMS =
-      new File(getClass().getResource("mtx/msOptional.tsv").getFile());
+//  private File withoutMS =
+//      new File(getClass().getResource("mtx/msOptional.tsv").getFile());
```
[the remark this answers is not in the document]
Comment (wow-such-code) [reply src/test/java/life/qbic/expdesign/io/MetaboDesignReaderTest.java]: good point, I rolled the change back by adding the missing test file

**Review by KochTobi — APPROVED** [review]

**Pull request merged**